chatbot-ui/components/PageFooter.test.tsx

```diff
@@ -2,12 +2,39 @@ import { render, screen } from '@testing-library/react';
 import PageFooter from '@/components/PageFooter';
 
 describe('Page Footer', () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...OLD_ENV };
+  });
+
+  afterAll(() => {
+    process.env = OLD_ENV;
+  });
+
   it('Renders', () => {
     render(<PageFooter />);
 
     expect(screen.getByRole('contentinfo')).toBeInTheDocument();
   });
 
+  it('Displays the build number if one is present', () => {
+    process.env.NEXT_PUBLIC_BUILD_NUMBER = 'Testing123';
+
+    render(<PageFooter />);
+
+    expect(screen.getByText('Build number: Testing123')).toBeInTheDocument();
```

Comment:
I think it's my fault, but this is breaking the UI unit tests since we changed it to `Build: 123` rather than `Build number 123`. Happy to update it myself...